hal/targets/hal/TARGET_ONSEMI/TARGET_NCS36510/spi_api.c

```diff
@@ -59,24 +59,28 @@ void spi_free(spi_t *obj)
 
 void spi_format(spi_t *obj, int bits, int mode, int slave)
 {
-    if(slave) {
+    if(slave) 
+	{
```

Comment:
Could you please revert this change to the bracket? It does not fit with our style guide: https://github.com/ARMmbed/mbed-os/blob/master/docs/COMMITTERS.md#c-coding-rules--coding-guidelines